tests/test_api/test_items.py

```diff
@@ -200,6 +200,29 @@ def test_get_items(self, test_client, user_item_saved):
         data = json.loads(r.data)
         assert len(data['items']) == 1
 
+class TestTrendingItemListAPI(TestBase):
+
+    def test_get_items_only_nonzero_votes(self, test_client, user_item_upvote, user_item):
+        self.login(test_client, user_item_upvote.user.email, user_item_upvote.user.password)
+        r = test_client.get('/api/items/trending')
+
+        check_valid_header_type(r.headers)
+        assert r.status_code == 200
+
+        data = json.loads(r.data)
+        assert len(data['items']) == 1
+
+    def test_get_items_missing(self, test_client, user):
```

Comment:
I was using unzipping to unpack (item, count) tuples. We don't do that anymore. 

Feel free to delete it here and merge if you're ready. Or leave it, either way lol